docker-compose.yml

```diff
@@ -1,40 +1,20 @@
-version: '2.2'
-
+volumes:
+  postgres_data_local: {}
```

Comment:
I don't totally understand what's going on under the hood here. [It has something to do](https://docs.docker.com/compose/compose-file/compose-file-v3/#volume-configuration-reference) with naming a volume before mounting it to `/var/lib/postgresql/data` but I am not sure 🤷